.github/workflows/ci.yml

```diff
@@ -24,3 +24,31 @@ jobs:
         uses: actions/checkout@v2
       - name: Run tests
         run: sbt -v +test
+  deploy:
+    name: Package and Deploy
+    needs: test
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 8
+      - name: Package with sbt-native-packager
+        run: sbt dist
+      - name: Configure AWS credentials for GitHub OIDC
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          role-to-assume: arn:aws:iam::495195014394:role/IAMGitHubOIDCRole
+          aws-region: eu-west-2
+      - name: Copy package to S3 bucket
+        working-directory: target/universal
+        run: aws s3 cp *.zip s3://ctd-omega-frontend-deployment/
+      - name: Deploy package to EC2
+        run: aws deploy create-deployment --application-name CtdOmegaEditorialFrontend --deployment-group-name CtdOmegaEditorialFrontend-DepGrp --revision revision-type=S3 --s3-location bucket=ctd-omega-frontend-deployment,key=ctd-omega-editorial-frontend-0.1.0-SNAPSHOT.zip,bundleType=zip
```

Comment:
The version (`ctd-omega-editorial-frontend-0.1.0-SNAPSHOT.zip`) is hardcoded. 

Is this temporary?

Reply:
Yes, this isn't ideal. Either we have to skip version numbers and always use the same package name or we can try to attach the version number when the GitHub Actions run. I think this could be acheived by serailizing the BuildInfo to JSON and then parsing the JSON within the workflow. Not trivial though so I think we would need another ticket for that.

Reply:
Agreed; could we introduce a placeholder ticket for that work (just a minimal description)?